docs/modules/reference/pages/daemons/introduction.adoc

```diff
@@ -0,0 +1,50 @@
+= List of daemons
+
+.List of daemons
```

Comment:
I think having the table name is redundant, given it's the only thing on the page, and is related to the title.
```suggestion
```